samcli/commands/_utils/custom_options/replace_help_option.py

```diff
@@ -0,0 +1,12 @@
+import click
+
+
+class ReplaceHelpSummaryOption(click.Option):
+    def __init__(self, *args, **kwargs):
+        self.replace_help_option = kwargs.get("replace_help_option", "")
+        kwargs.pop("replace_help_option")
```

Comment:
nit: Does this work?
```suggestion
        self.replace_help_option = kwargs.pop("replace_help_option", "")
```

Reply:
I'll try it out! Thanks for the suggestion!